.travis.yml

```diff
@@ -7,4 +7,5 @@ install:
   - pip install -r requirements/ci.txt
 
 script:
-    - make test
\ No newline at end of file
+    - make test
+    - make docker
```

Comment:
`make docker` would be better _[before_deploy](https://docs.travis-ci.com/user/customizing-the-build/)_.

And later, this project can also set a _deploy_ task, to deploy the image on the server somehow.